multiple-search-services/README.md

```diff
@@ -53,6 +53,12 @@ Open the **appsettings.json** file in your local copy of the sample application
 
     + Find the URI in the [search service's Overview portal page](https://docs.microsoft.com/azure/search/search-manage#overview-home-page).
 
+1. "semanticConfigurationName": "Semantic Configuration Name on your index":
+
+    + Required if you want to request a Semantic query. Find the Semantic Config in the [search service's desired index in the search explorer under the Configurations tab](https://learn.microsoft.com/en-us/azure/search/semantic-how-to-query-request?tabs=portal%2Cportal-query).
+
+    **Note**: You must create a Semantic Configuration first in the Portal or via the REST API before issuing a Semantic query. Additionally, you can find the list of query languages in the [List of supported languages](https://learn.microsoft.com/en-us/rest/api/searchservice/preview-api/search-documents#queryLanguage)
```

Comment:
line 60 - if you accept the above suggestion, you can scope this note to just the query language consideration. "Semantic ranking comes with language requirements. See [List of supported language](https://learn.microsoft.com/rest/api/searchservice/preview-api/search-documents#queryLanguage) for details.

Reply:
Done, please review.